test/integration/features/addons/addon-che-with-anyuid.feature

```diff
@@ -0,0 +1,72 @@
+@addon-che-with-anyuid @addon @core
```

Comment:
@agajdosi `@core` means it will run with each PR. Is it is a temporary tag to see if the test runs fine or not?

Reply:
Yes, this is temporary, will be removed with [WIP] tag.